packages/authorization-process/src/routers/AuthorizationRouter.ts

```diff
@@ -553,6 +555,33 @@ const authorizationRouter = (
       }
     });
 
+  if (config.featureFlagRemoveClientAdmin) {
+    authorizationClientRouter.delete(
+      "/clients/:clientId/admin/:adminId",
+      async (req, res) => {
+        const ctx = fromAppContext(req.ctx);
```

Comment:
```suggestion
    authorizationClientRouter.delete(
      "/clients/:clientId/admin/:adminId",
      async (req, res) => {
      if (!config.featureFlagAdminClient) {
        return res.status(501).send({
          title: "Feature not implemented",
          detail: "The client admin removal feature is currently disabled",
        });
      }
      
        const ctx = fromAppContext(req.ctx);
```
I would do something like this, but in the BFF, you could also extract that logic outside in order to reuse it in all the endpoints that require it